firmware_configurations/klipper/redelman/Spider_2.2_Voron_Trident/Trident_Spider_2_2.cfg

```diff
@@ -0,0 +1,483 @@
+
+speed: 200
+horizontal_move_z: 10
+retries: 5
+retry_tolerance: 0.0075
+
+#####################################################################
+#       Displays
+#####################################################################
+
+[display]
+# mini12864 LCD Display
+lcd_type: uc1701
+cs_pin: PC11
+a0_pin: PD2
+rst_pin: PC10
+encoder_pins: ^PC6,^PC7
+click_pin: ^!PA8
+contrast: 63
+#spi_bus: spi1
+spi_software_mosi_pin: PA7
+spi_software_miso_pin: PA6
+spi_software_sclk_pin: PA5
+
+[neopixel fysetc_mini12864]
+# To control Neopixel RGB in mini12864 display
+pin: PC12
+chain_count: 3
+initial_RED: 0.1
+initial_GREEN: 0.5
+initial_BLUE: 0.0
+color_order: RGB
+
+# Set RGB values on boot up for each Neopixel.
+# Index 1 = display, Index 2 and 3 = Knob
+[delayed_gcode setdisplayneopixel]
+initial_duration: 1
+gcode:
+        SET_LED LED=fysetc_mini12864 RED=1 GREEN=1 BLUE=1 INDEX=1 TRANSMIT=0
+        SET_LED LED=fysetc_mini12864 RED=1 GREEN=0 BLUE=0 INDEX=2 TRANSMIT=0
+        SET_LED LED=fysetc_mini12864 RED=1 GREEN=0 BLUE=0 INDEX=3
+
+
+#####################################################################
+#   Macros
+#####################################################################
+
+[gcode_macro PRINT_START]
+# Use PRINT_START for the slicer starting script - PLEASE CUSTOMISE THE SCRIPT
+gcode:
+    M117 Homing...                 ; display message
+    G28
+    Z_TILT_ADJUST
+    G28
+
+    ##  Uncomment for for your size printer:
+    #--------------------------------------------------------------------
+    ##  Uncomment for 250mm build
+    #G0 X125 Y125 Z30 F3600
+
+    ##  Uncomment for 300 build
+    #G0 X150 Y150 Z30 F3600
+
+    ##  Uncomment for 350mm build
+    #G0 X175 Y175 Z30 F3600
+    #--------------------------------------------------------------------
+
+   
+
+[gcode_macro PRINT_END]
+#   Use PRINT_END for the slicer ending script - please customise for your slicer of choice
+gcode:
+    # safe anti-stringing move coords
+    {% set th = printer.toolhead %}
+    {% set x_safe = th.position.x + 20 * (1 if th.axis_maximum.x - th.position.x > 20 else -1) %}
+    {% set y_safe = th.position.y + 20 * (1 if th.axis_maximum.y - th.position.y > 20 else -1) %}
+    {% set z_safe = [th.position.z + 2, th.axis_maximum.z]|min %}
+    
+    SAVE_GCODE_STATE NAME=STATE_PRINT_END
+    
+    M400                           ; wait for buffer to clear
+    G92 E0                         ; zero the extruder
+    G1 E-2.0 F3600                 ; retract filament
+    
+    TURN_OFF_HEATERS
+    
+    G90                                      ; absolute positioning
+    G0 X{x_safe} Y{y_safe} Z{z_safe} F20000  ; move nozzle to remove stringing
+    G0 X{th.axis_maximum.x//2} Y{th.axis_maximum.y - 2} F3600  ; park nozzle at rear
+    M107                                     ; turn off fan
+    
+    BED_MESH_CLEAR
+    RESTORE_GCODE_STATE NAME=STATE_PRINT_END
+
+##   Common Temperature Sensors
```

Comment:
Can you remove this section? some of these thermistor types have been removed, and the link in the comments above should be enough.

Reply:
Yup, consider it done.